src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -0,0 +1,32 @@
+package com.courtcanva.ccfranchise.controllers;
+
+import com.courtcanva.ccfranchise.dtos.FranchiseeDto;
+import com.courtcanva.ccfranchise.services.FranchiseeService;
+import com.courtcanva.ccfranchise.services.StaffService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping
+public class FranchiseeController {
+
+    @Autowired
```

Comment:
If you look at this class on your IDE, you would find a warning with this annotation that injecting bean with @Autowired is deprecated now. Pleas use constructor injection instead.

Reply:
thx, lao